runbook/executor.go

```diff
@@ -58,8 +78,19 @@ func Execute(executablePath string, args, environmentVars []string, stdout, stde
 
 	err := cmd.Run()
 	if err != nil {
-		return &ExecError{stderrBuff.String(), err}
+		return "", &ExecError{stderrBuff.String(), err}
 	}
 
-	return nil
+	waitGroup.Done()
```

Comment:
We should not need a wait group here since running the script is a synchronous operation